fix: Updated dependencies to resolve build warning

swapnilsparsh opened the pull request:
1. To resolve the issue, we needed to update a few dependencies, including `electron-vite <=3.0.0` Since the only available version greater than this was a beta release published yesterday, I installed that to address the warning.

2. Removed the unused `protocol` import as it was giving warning during build.

kocmo commented:
@swapnilsparsh - I also get notices about outdated node modules when I first compile in a new environment:

```
*** Installing NPM molules ... ***                                                                                                                                                                                                            
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and power
ful.                                                                                                                                                                                                                                          
npm warn deprecated @humanwhocodes/config-array@0.11.14: Use @eslint/config-array instead                                                                                                                                                     
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported                                                                                                                                                         
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported                                                                                                                                                             
npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead

```
Could you please look at these also?

swapnilsparsh commented:
@kocmo
I have updated the dependencies to their latest versions. However, some warnings persist due to `electron-builder` despite using the latest version available.

We can update `@sentry/electron` and `sass` to their latest versions, but some features have been deprecated and are incompatible with our current code. Updating these packages would require additional changes, which is why they have not been updated at this time.

Additionally, I have fixed the login screens to ensure proper display on smaller window sizes.

kocmo commented:
Hi @swapnilsparsh:

> We can update @sentry/electron 

Do we still need Sentry?... Let me check the `feat/plcon-65-submit-logs-to-rageshake` branch (not pushed yet), how much we need of Sentry functionality still - and let's disable the rest of Sentry.

> ... and sass

Yes, sass warnings in attached build log on Debian 11 (our build env for Linux), let's discuss further steps.

[PR-65.pl_connect_build_log.sass_warning.log](https://github.com/user-attachments/files/19304476/PR-65.pl_connect_build_log.sass_warning.log)
